Tests/NetworkServiceTests/URLParametersEncoderTests.swift

```diff
@@ -0,0 +1,68 @@
+import NetworkService
+import XCTest
+
+final class URLParametersEncoderTests: XCTestCase {
+    let testURL = Bundle.main.bundleURL
+    var request: URLRequest!
+
+    override func setUp() {
+        super.setUp()
+        request = URLRequest(url: testURL)
+    }
+
+    override func tearDown() {
+        super.tearDown()
+        request = nil
+    }
+
+    func testQueryItemsPropertyOfRequestIsNilWhenNilIsPassedAsParameters() throws {
+        try assertQueryItemsEquality(parameters: nil)
+    }
+
+    func testQueryItemsPropertyOfRequestIsEmptyArrayWhenEmptyDictionaryIsPassedAsParameters() throws {
+        try assertQueryItemsEquality(parameters: [:])
+    }
+
+    func testQueryItemsPropertyOfRequestHasExpectedValueWhenPassedOneKeyValuePairOfURLParameters() throws {
```

Comment:
Maybe you could simplify the name of the tests or the meaning in this case could lost?
```suggestion
    func testQueryItemsPropertyHasSingleItemWhenSingleKeyValuePairPassed() throws {
```

Reply:
Your proposal doesn't explain the test's meaning. It doesn't just check the amount of query Items, but it checks their equality to expected values as well.